druid-derive/src/attr.rs

```diff
@@ -22,6 +22,7 @@ use syn::{Error, ExprPath, Meta, NestedMeta};
 const BASE_ATTR_PATH: &str = "druid";
 const IGNORE_ATTR_PATH: &str = "ignore";
 const SAME_FN_ATTR_PATH: &str = "same_fn";
+const LENS_NAME_OVERRIDE_ATTR_PATH: &str = "lens_name";
```

Comment:
I feel like in the future we might want to replace the `#[druid]` attribute with separate `#[lens]` and `#[data]` attributes, but I think this is reasonable for now.